apps/src/aiTutor/views/AITutorFooter.tsx

```diff
@@ -134,7 +135,9 @@ const AITutorFooter: React.FC<AITutorFooterProps> = ({renderAITutor}) => {
         {showCompilationOption && (
           <Button
             className={style.quickActionButton}
-            color={Button.ButtonColor.teal}
+            color={buttonColors.gray}
+            type={'secondary'}
+            size="s"
```

Comment:
I'm not too worried about the side effect style changes to this button and the validation button because the [plan ](https://www.figma.com/design/lOLzGeTfH44a5YDS7GrzdA/AI-Tutor?node-id=2386-8167&t=I89XwOK2y2MlJ50n-0) is to move them into the chat as suggested prompts.